crates/tuono/src/env.rs

```diff
@@ -0,0 +1,176 @@
+use crate::mode::Mode;
+use std::collections::HashSet;
+use std::env;
+use std::fs;
+
+#[derive(Clone, Debug)]
+pub struct EnvVarManager {
+    env_files: Vec<String>,
+    mode: Option<Mode>,
+    system_env: HashSet<String>,
+}
+
+impl EnvVarManager {
+    pub fn new(mode: Option<Mode>) -> Self {
+        let mut env_files = vec![String::from(".env"), String::from(".env.local")];
+
+        if let Some(mode) = &mode {
+            let mode_name = match mode {
+                Mode::Dev => "development",
+                Mode::Prod => "production",
+            };
```

Comment:
We might have multiple environments.
I think the user could just define `PROJECT_MODE` variable in its own `.env`s to deal with this

Reply:
mmm from the doc looks like there are just two mode `production` and `development`. Maybe more are not necessary!

I'm not sure about `NODE_ENV` since we are not using node... It might be misleading.